src/components/molecules/Table/Table.jsx

```diff
@@ -26,64 +29,50 @@ const StyledTd = styled.td`
-        </tr>
-        <tr>
-          <StyledTd>My first project</StyledTd>
-          <StyledTd>Its only for testing</StyledTd>
-          <StyledTd>about 1 month ago</StyledTd>
-          <StyledTd>
-            <Button label="Tasks" onClick={test} disabled={false} color="thistle" />
-          </StyledTd>
-          <StyledTd>
-            <Button label="Edit" onClick={test} disabled={false} color="thistle" />
-          </StyledTd>
-          <StyledTd>
-            <Button label="Destroy" onClick={test} disabled={false} color="thistle" />
-          </StyledTd>
-        </tr>
-        <tr>
-          <StyledTd>My first project</StyledTd>
-          <StyledTd>Its only for testing</StyledTd>
-          <StyledTd>about 1 month ago</StyledTd>
-          <StyledTd>
-            <Button label="Tasks" onClick={test} disabled={false} color="thistle" />
-          </StyledTd>
-          <StyledTd>
-            <Button label="Edit" onClick={test} disabled={false} color="thistle" />
-          </StyledTd>
-          <StyledTd>
-            <Button label="Destroy" onClick={test} disabled={false} color="thistle" />
-          </StyledTd>
-        </tr>
-        <tr>
-          <StyledTd>My first project</StyledTd>
-          <StyledTd>Its only for testing</StyledTd>
-          <StyledTd>about 1 month ago</StyledTd>
-          <StyledTd>
-            <Button label="Tasks" onClick={test} disabled={false} color="thistle" />
-          </StyledTd>
-          <StyledTd>
-            <Button label="Edit" onClick={test} disabled={false} color="thistle" />
-          </StyledTd>
-          <StyledTd>
-            <Button label="Destroy" onClick={test} disabled={false} color="thistle" />
-          </StyledTd>
-        </tr>
-      </tbody>
-    </StyledTable>
+    <>
+      {!loading && (
+        <StyledTable>
+          <tbody>
+            <tr>
+              <StyledTh>Name</StyledTh>
+              <StyledTh>Description</StyledTh>
+              <StyledTh>Created at</StyledTh>
+              <StyledTh>&nbsp;</StyledTh>
+              <StyledTh>&nbsp;</StyledTh>
+              <StyledTh>&nbsp;</StyledTh>
+            </tr>
+            {projects.map(({ name, description, createdAt, id }) => (
+              <tr>
+                <StyledTd>{name}</StyledTd>
+                <StyledTd>{description}</StyledTd>
+                <StyledTd>{new Date(createdAt).toLocaleDateString("RU-ru")}</StyledTd>
+                <StyledTd>
+                  <Button label="Tasks" onClick={() => routeChange({ id })} color="thistle" />
+                </StyledTd>
+                <StyledTd>
+                  <Button label="Edit" onClick={test} color="thistle" />
+                </StyledTd>
+                <StyledTd>
+                  <Button label="Destroy" onClick={test} color="thistle" />
+                </StyledTd>
+              </tr>
+            ))}
+          </tbody>
+        </StyledTable>
+      )}
+      <div>{loading && <h1>LOADING NOW</h1>}</div>
```

Comment:
dev unneeded, it shouldn't render after loading
```suggestion
      {loading && <h1>LOADING NOW</h1>}
```
<img width="318" alt="Screenshot 2023-04-24 at 15 54 45" src="https://user-images.githubusercontent.com/51396910/234002398-c78fa584-3fe7-44ff-9ace-007b9a6c7769.png">